.gitlab-ci.yml

```diff
@@ -119,16 +191,16 @@ centos8:
     - yum install -y https://github.com/stewartsmith/libeatmydata/releases/download/v129/libeatmydata-129-1.fc33.x86_64.rpm
     - yum install -y ccache  # From EPEL
     - source /etc/profile.d/ccache.sh
-    - export CCACHE_DIR="$(pwd)/.ccache"; ccache -s
+    - export CCACHE_DIR="$(pwd)/.ccache"; ccache -z
```

Comment:
Not a stopper: I had to look up "ccache -z", can we make this "ccache --zero-stats" maybe?

Reply:
Done